src/lib/types/Currency.ts

```diff
@@ -41,6 +41,32 @@ export const FRACTION_DIGITS_PER_CURRENCY = Object.freeze({
 	CZK: 2
 }) satisfies Record<Currency, number>;
 
+/**
+ * Maximum number of decimal places for storing prices WITHOUT VAT.
+ * This allows merchants to set precise base prices that result in round
+ * numbers after VAT calculation (e.g., 4.6253 CHF → 5.00 CHF with 8.1% VAT).
+ *
+ * Display precision remains FRACTION_DIGITS_PER_CURRENCY (2 for most currencies).
+ */
+export const STORAGE_FRACTION_DIGITS_PER_CURRENCY = Object.freeze({
```

Comment:
This is a representation detail and must be persisted alongside with the price.

If we change it in the future, the interpretation of whatever we stored in the database should not depend on a code constant; else we'll have to implement heuristics to guess what we meant to store or break backwards compatibility.

We need to store the price precision along with the amount currency and value.

It would be useful to define a "currency amount" type that contains the currency, value and precision.